live-engage.android.ts

```diff
@@ -119,4 +119,25 @@ export class LiveEngage implements CommonLiveEngage {
     public parsePushMessage(data: any): any {
         return new com.liveperson.infra.messaging_ui.uicomponents.PushMessageParser(data);
     }
+
+    public killChat(): void {
+        if (!this.brandId || !this.appId) {
+            return;
+        }
+
+        if (!this.isValidState()) {
+            return;
+        }
+
+        com.liveperson.messaging.MessagingFactory.getInstance().getController().resolveConversation(this.brandId, this.brandId);
+        const LogoutCallback: any = com.liveperson.infra.callbacks.LogoutLivePersonCallBack.extend({
+            onLogoutSucceed: () => { },
+            onLogoutFailed: (err: any) => {
+                console.error(err);
+            }
+        });
+        const initProperties = new com.liveperson.infra.InitLivePersonProperties(this.brandId, this.appId, null);
+        const ui = new com.liveperson.infra.messaging_ui.MessagingUiInitData(initProperties, this.getSDKVersion());
+        com.liveperson.infra.messaging_ui.MessagingUIFactory.getInstance().logout(application.android.context, ui, new LogoutCallback());
```

Comment:
Logging out means the instance will not run anymore and has to be reinitialised next time. Is that the only way to get rid of the customer data or would be clearing the history good enough? Secondly, what happens to registered push notifications, are these unregistered as well?